stream_chat/base/client.py

```diff
@@ -26,6 +28,24 @@ def __init__(
             {"server": True}, self.api_secret, algorithm="HS256"
         )
 
+    def _enrich_response(
+        self,
+        response: Union[aiohttp.ClientResponse, requests.Response],
+        resp_body: StreamResponse,
+    ) -> StreamResponse:
+        resp_headers: Dict[str, Any] = dict(response.headers)  # type: ignore
+
+        resp_body["rate_limit"] = dict(
```

Comment:
how sure are we these headers are always present?

Reply:
@gumuz refactored it